Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -564,69 +566,66 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     // MARK: - Tunnel Start
 
-    open override func startTunnel(options: [String: NSObject]?, completionHandler: @escaping (Error?) -> Void) {
-        Task { @MainActor in
-            providerEvents.fire(.tunnelStartAttempt(.begin))
-            prepareToConnect(using: tunnelProviderProtocol)
```

Comment:
As `prepareToConnect` is necessary to set up PixelKit, we might've undercounted `tunnel-start-attempt-begin` 🤔

Reply:
I think we're fine, but the existing logic is a little redundant. `prepareToConnect` does indeed set up pixels, but **only** if they haven't already been set up, and thankfully we also call `setupPixels` during packet tunnel provider init. I'd guess that the set-up call within `prepareToConnect` never runs because it's always set up at this point.

Reply:
I think there's merit to this observation.

The call to `loadVendorOptions` is setting up PixelKit but if can fail before it's set up.  I think this call is currently needed, and moving it to the top seems aligned with the spirit of it, so for simplicity I'll do that.

We can review this code at a second time, but moving this to the top should be fairly safe.